Jumpscale/clients/blockchain/tfchain/schemas/reservation_s3.schema

```diff
@@ -0,0 +1,6 @@
+@url = tfchain.reservation.s3
+type = "s3" (S)
+duration = (I)
```

Comment:
duration is still defined here, but I do not see it being used, is that OK?

Reply:
yes I'm planning for the future, that's fine